### PR TITLE
261 new color scheme

### PR DESCRIPTION
**Pull request by radscheit**

### CODE REVIEW

**Review by phnm — COMMENTED** [review]
- color scheme (red/green/gray) of agreement letter state on the participants overview page (/events/id/participants) is gone; it's just white now and hard to read.
- 'register now' button on landing page doesn't link correctly.
- I think it would be nicer if the red color of the delete buttons would be the hpi red. it's not so in-line with the color scheme now.
- heading on print badges page (/events/id/badges) is way too big.

File: Gemfile.lock [hunk 1]
```diff
@@ -256,7 +256,7 @@ GEM
     slop (3.6.0)
     spring (2.0.0)
       activesupport (>= 4.2)
-    sprockets (3.7.0)
+    sprockets (3.6.3)
```
Comment (bjrne) [inline Gemfile.lock:259]: Is this downgrade intentional?

**Review by bjrne — COMMENTED** [review]
From what I can say, the code should be fine. But when I tested this locally, the response time on a complete reload of the page was 5+ seconds. I doubt that this is acceptable. Switching subpages seemed unaffected and still happens in the blick of an eye.

**Comment by bjrne** [comment]
I got you a log from what happened in a single reload on my machine. Maybe you can look into it and figure out what went wrong: [log.txt](https://github.com/hpi-swt2/workshop-portal/files/713888/log_new_design.txt)

**Comment by bjrne** [comment]
And I feel like the "deletebutton"-red is a tiny bit too pink and to bright, it doesn't really match the calm  orange.

**Comment by corinnaj** [comment]
The picture overlap the text for some screen sizes, see picture.
screenshot from 2017-01-18 15-12-18

**Comment by bjrne** [comment]
And there are two "Willkommen am HPI" parts, but that may change with correct text examples. I'm just not sure whether that is still intended. (One at the top and one near the bottom)

**Comment by tom95** [comment]
* Fixed the picture thing being weird
* Changed the red to HPI "red"
* The second block is supposed to house an "about us" section later on, as seen in #350

**Review by corinnaj — APPROVED** [review]
Looks good to me.

**Comment by phnm** [comment]
- the text color (red/green/gray) of agreement letter state on the participants overview page (/events/id/participants) is still gone
grafik

- for some reason the status selection of applications on the event's page doesn't save anymore

**Comment by Baschdl** [comment]
@radscheit @tom95 You broke the applicant selection with this PR. Could you fix it? 
pasted_image_at_2017_01_19_05_48_pm_360

**Comment by tom95** [comment]
@Baschdl MR #358 fixes it and adds tests to check if it's working

**Pull request merged**